Source/Pads/Keyboard.as

```diff
@@ -91,15 +117,26 @@ class DashboardPadKeyboard : IDashboardPad
 		}
 		nvg::Stroke();
 
+		vec4 textColor = Setting_Keyboard_FontColor;
+		textColor.w *= fillAlpha;
+
 		nvg::BeginPath();
-		nvg::FontFace(g_font);
-		nvg::FontSize(size.x / 2);
-		if (Setting_Keyboard_UseBorderGradient) {
-			nvg::FillPaint(Setting_Keyboard_BorderGradient.GetPaint(vec2(), m_size, fillAlpha));
-		} else {
-			nvg::FillColor(borderColor);
-		}
```

Comment:
This was intentional (gradient matching the border), and it appears there's no replacement for this now. I understand why you did this, but I'm not sure it's the right move.